docs/content/preview/integrations/keycloak.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Keycloak
+linkTitle: Keycloak
+description: Use Keycloak with YSQL API
+aliases:
+menu:
+  preview_integrations:
+    identifier: keycloak
+    parent: integrations-security
+    weight: 571
+type: docs
+---
+
+[Keycloak](https://www.keycloak.org/) is an open source identity and access management tool that adds authentication to applications and secure services with minimum effort.
```

Comment:
Why am I using keycloak with YBDB though? What is the benefit/point

Reply:
Keycloak is an authentication tool that you can use with your applications. It stores the username and password in a database, we are adding this document to configure that database to be YugabyteDB